packages/core/upload/admin/src/components/Breadcrumbs/tests/index.test.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { render as renderTL, screen, fireEvent, waitFor } from '@testing-library/react';
+import { QueryClientProvider, QueryClient } from 'react-query';
+import { MemoryRouter } from 'react-router-dom';
+
+import { ThemeProvider, lightTheme } from '@strapi/design-system';
+import { Breadcrumbs } from '../index';
+import en from '../../../translations/en.json';
+
+jest.mock('../../../hooks/useFolderStructure');
+
+jest.mock('../../../utils', () => ({
+  ...jest.requireActual('../../../utils'),
+  getTrad: x => x,
+}));
+
+jest.mock('@strapi/helper-plugin', () => ({
+  ...jest.requireActual('@strapi/helper-plugin'),
+  useQueryParams: jest.fn().mockReturnValue([{ query: { folder: 22 } }]),
+}));
+
+jest.mock('react-intl', () => ({
+  useIntl: () => ({ formatMessage: jest.fn(({ id }) => en[id]) }),
+}));
+
+const queryClient = new QueryClient({
+  defaultOptions: {
+    queries: {
+      retry: false,
+      refetchOnWindowFocus: false,
+    },
+  },
+});
+
+const breadcrumbs = [
+  {
+    href: '/',
+    id: null,
+    label: 'Media Library',
+  },
+  [],
+  { href: '/', id: 21, label: 'parent folder' },
+  { id: 22, label: 'current folder' },
+];
+
+const setup = props =>
+  renderTL(
+    <QueryClientProvider client={queryClient}>
+      <MemoryRouter>
+        <ThemeProvider theme={lightTheme}>
+          <Breadcrumbs {...props} breadcrumbs={breadcrumbs} as="nav" />
```

Comment:
Could you please pass `breadcrumbs` per test, so that it is easier in the future to add new ones?